File: Java/백준/Silver/1436. 영화감독 숌/영화감독 숌.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine().trim());
        int count = 0;
        int num = 666;
        while (true) {
            if (String.valueOf(num).contains("666")) {
                count++;
                if (count == N) {
                    System.out.println(num);
                    break;
                }
            }
            num++;
        }
    }
}

File: Python/백준/Silver/1436. 영화감독 숌/영화감독 숌.py
def find_nth_movie(n):
    count = 0
    num = 666
    while True:
        if "666" in str(num):
            count += 1
            if count == n:
                return num
        num += 1

if __name__ == "__main__":
    import sys
    input = sys.stdin.readline
    N = int(input().strip())
    print(find_nth_movie(N))